webviz_subsurface/plugins/_segy_viewer.py

```diff
@@ -196,43 +198,48 @@ def settings_layout(self):
         )
 
     @property
-    def plot_layout(self):
-        """Layout for main plots"""
+    def layout(self):
         return html.Div(
-            style=self.set_grid_layout("2fr 1fr"),
-            children=[
-                html.Div(
-                    children=[
-                        html.Div(
-                            style={"height": "400px"},
-                            children=wcc.Graph(
-                                config={"displayModeBar": False}, id=self.ids("inline")
+            id=self.ids("layout"),
+            children=wcc.FlexBox(
+                children=[
+                    html.Div(
+                        style={"width": "50%"},
+                        children=[
+                            html.Div(
+                                style={"minWidth": "200px", "height": "400px"},
+                                children=wcc.Graph(
+                                    config={"displayModeBar": False},
+                                    id=self.ids("inline"),
+                                ),
                             ),
-                        ),
-                        html.Div(
-                            style={"height": "400px"},
-                            children=wcc.Graph(
-                                config={"displayModeBar": False}, id=self.ids("xline")
+                            html.Div(
+                                style={"minWidth": "200px", "height": "400px"},
+                                children=wcc.Graph(
+                                    config={"displayModeBar": False},
+                                    id=self.ids("xline"),
+                                ),
                             ),
-                        ),
-                    ]
-                ),
-                html.Div(
-                    style={"height": "876px"},
-                    children=wcc.Graph(
-                        config={"displayModeBar": False}, id=self.ids("zslice")
+                        ],
                     ),
-                ),
-                dcc.Store(
-                    id=self.ids("state-storage"), data=json.dumps(self.init_state)
-                ),
-            ],
-        )
-
-    @property
-    def layout(self):
-        return html.Div(
-            id=self.ids("layout"), children=[self.settings_layout, self.plot_layout]
+                    html.Div(
+                        style={"height": "876px", "width": "50%"},
```

Comment:
This height value looks special. Any reason behind that particular value? If so, add it as a comment?